### DISCUSSION TITLE
Import/Download missing

### DISCUSSION THREAD

**Discussion by neilbingham** [WeblateOrg/weblate#8666] [Q&A] [answered]
I've setup a new deployment using 4.16 and started experimenting with Weblate.

When I setup a new project and component I wanted to look at the download and import features.  What I'm seeing does not match the documentation.  I do have Translate rights for the project, but I only get a Download as Zip option on the Files menu which gives me the JSON files I configured as the output.  

The [manual](https://docs.weblate.org/en/latest/user/files.html) says I should be able to download CSV etc. for offline editing and then import again.

It doesn't show up as a Super User either.  What have I missed?

Thanks,

Neil.

**Comment by nijel** [comment, marked as answer]
You need to go one level deeper - to the translation, you will get more download/upload options there.

**Reply by neilbingham** [reply]
Ah I see, yes that works thanks.

Shame you can't export a zip of (for example) CSV files at the component level.  That would be useful.

Neil.

**Reply by nijel** [reply]
I've created https://github.com/WeblateOrg/weblate/issues/8680 to track this.
